index.html

```diff
@@ -4446,27 +4447,27 @@ <h3>Verification</h3>
         <p>
 This section contains an algorithm that <a>conforming verifier
 implementations</a> MUST run when verifying a <a>verifiable credential</a> or a
-<a>verifiable presentation</a>. This algorithm takes a byte sequence (bytes
-<var>inputBytes</var>) and a media type (string <var>inputMediaType</var>) as
-inputs, and produces an object (<var>result</var>) that contains the
-following:
+<a>verifiable presentation</a>. This algorithm takes a sequence of bytes
+([=byte sequence=] <var>inputBytes</var>) and a media type
```

Comment:
My concern w/ `[=MIME type=]` is that I'm not sure that developers implementing this specification type media types in that way (they probably just treat them as strings). I don't know if it will cause them to think that the check is going to be anything other than a straight string compare (in most implementations). At least, that's my current thinking wrt. `inputMediaType` being a string. Happy to be swayed another way if there is a stronger argument for stronger typing.